lm_eval/tasks/mathqa.py

```diff
@@ -0,0 +1,48 @@
+from . common import HFTask
+from lm_eval.base import mean, rf, MultipleChoiceTask
+
+
+class MathQA(HFTask, MultipleChoiceTask):
+    DATASET_PATH = "math_qa"
+    DATASET_NAME = None
+
+    def has_training_docs(self):
+        return True
+
+    def has_validation_docs(self):
+        return True
+
+    def has_test_docs(self):
+        return True
+
+    def _convert_standard(self, doc):
+
+        out_doc = {
+            "query": "Question: " + doc['Problem'] +" "+ doc["options"] + "\nAnswer:",
+            "choices": ['a', 'b', 'c', 'd', 'e'],
```

Comment:
`choices` should not be the literal letter answer but the text associated with the answer. E.g. for something like 
`a) 185 ,b) 204 ,c) 203 ...` the choices should be the list `[185, 204, 203...]`.